.circleci/config.yml

```diff
@@ -0,0 +1,100 @@
+version: 2.1
+orbs:
+  node: circleci/node@5.0.2
+jobs:
+  build:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
```

Comment:
this step looks like it should `npm install` for you already. What does it do?

Reply:
you're right, it takes care of doing npm install already.